Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/photo_field.tsx

```diff
@@ -0,0 +1,653 @@
+import { css, StyleSheet } from 'aphrodite/no-important';
+import * as React from 'react';
+import { Transition } from 'react-transition-group';
+import { DisplaySize, HBoxLayout, HLine, Padding, VBoxLayout } from '../../..';
+
+/** Various the mode that the PhotoField can be displayed at. */
+export enum DisplayMode {
+
+  /** Only the photo is visible. */
+  Display,
+
+  /** The uploader is visible. */
+  Uploading
+}
+
+interface Properties {
+
+  /** Determines the size to render the component at. */
+  displaySize: DisplaySize;
+
+  /** Determines if the ChangePictureModal is visible or not. */
+  displayMode: DisplayMode;
+
+  /** The URL the image is located at. */
+  imageSource?: string;
+
+  /** Determines if the image can be changed or not. */
+  readonly?: boolean;
+
+  /** A value from 0 to 200 that determines how zoomed in the image will be. */
+  scaling: number;
```

Comment:
I would prefer if this was a normalized scalar value, where 1.0 means no zoom is performed, 2.0 means twice as big, 0.5 means half as big, etc... Would that be possible?